config/gatekeeper/openshift/apps_v1_deployment_gatekeeper-audit.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
```

Comment:
I think it would be best to progammatically remove the annotations instead of duplicating the manifest, because otherwise it will create a larger maintenance burden to keep this file in sync.